backend/src/main/java/com/github/tonyglimm/media/Media.java

```diff
@@ -0,0 +1,17 @@
+package com.github.tonyglimm.media;
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+import java.util.List;
+
+public record Media(
+        int id,
+        @JsonAlias("original_name") String original_title,
+        List<Integer> genre_ids,
+        String poster_path,
+        @JsonAlias("first_air_date") String release_date,
+        String media_type,
+        float vote_average,
+        int vote_count,
+        String overview) {
```

Comment:
try to stick to the CamelCase convention

Reply:
Hmm. for the field names? Initially i named these like the fields in the JSON Object I recieve from the external API. Now that I know of @JSONAlias I could actually rename all of them to fields in CamelCase. Considering that do you still recommend renaming them and adding a @JsonAlias on each field?

Reply:
use instead @JsonProperty